extend.php

```diff
@@ -4,25 +4,40 @@
 
 use Blomstra\Gdpr\Api\Serializer\ExportSerializer;
 use Blomstra\Gdpr\Api\Serializer\RequestErasureSerializer;
+use Blomstra\Gdpr\Models\ErasureRequest;
 use Blomstra\Gdpr\Notifications;
+use Flarum\Api\Controller\ShowUserController;
+use Flarum\Api\Serializer\UserSerializer;
 use Flarum\Extend;
+use Flarum\User\User;
 
 return [
     (new Extend\Frontend('forum'))->js(__DIR__ . '/js/dist/forum.js'),
 
     (new Extend\Locales(__DIR__ . '/resources/locale')),
 
     (new Extend\Routes('forum'))
-        ->get('/gdpr/export/{file}', 'gdpr.export', Http\Controller\ExportController::class),
-        //->get('/gdpr/erasure/confirm/{token}', 'gdpr.erasure-confirm', NEWCONTROLLER),
+        ->get('/gdpr/export/{file}', 'gdpr.export', Http\Controller\ExportController::class)
+        ->get('/gdpr/erasure/confirm/{token}', 'gdpr.erasure.confirm', Http\Controller\ConfirmErasureController::class),
 
     (new Extend\Routes('api'))
         ->post('/gdpr/export', 'gdpr.request-export', Api\Controller\RequestExportController::class)
-        ->post('/gdpr/erase-me', 'gdpr.request-erasure', Api\Controller\RequestErasureController::class),
+        ->post('/user-erasure-requests', 'gdpr.erasure.create', Api\Controller\RequestErasureController::class)
+        ->delete('/user-erasure-requests/{id}', 'gdpr.erasure.cancel', Api\Controller\CancelErasureController::class),
 
     (new Extend\Notification)
         ->type(Notifications\ExportAvailableBlueprint::class, ExportSerializer::class, ['alert', 'email'])
-        ->type(Notifications\ConfirmErasureBlueprint::class, RequestErasureSerializer::class, ['email']),
+        ->type(Notifications\ConfirmErasureBlueprint::class, RequestErasureSerializer::class, ['alert', 'email'])
```

Comment:
```suggestion
        ->type(Notifications\ConfirmErasureBlueprint::class, RequestErasureSerializer::class, ['email'])
```
Do we want to generate an alert notification here? If the primary purpose of having this is to confirm the users' email address, would it not be wise to only offer the email version of the notification?